.config/forest.dic

```diff
@@ -99,3 +99,6 @@ VRF
 WebAssembly
 WebSocket
 zstd
+ParityDB
+benchmark/GD
++
```

Comment:
We need a plus sign in the dictionary?

```suggestion
```

Reply:
Yes, this is needed for e.g. `O(V+E)` comments.

Reply:
If you use back ticks (like you just did in your comment) then the spellchecker will leave it alone.

Reply:
Thanks, I did not know that.